full_coverage/SquareCell[] grid_cells
int32 grid_cols
time stamp
string frame_id
